From d0fcaec62110e093ed62a825897d454d61291a7d Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Mon, 22 Sep 2008 10:22:42 +0800
Subject: [PATCH 021/256] ENGR00092357 Fix imx-lib compiler error for imx31ads
 and imx32ads

Same PLATFORM ID is used when building VPU lib for imx31ads and imx32ads.
Change version number to imx32 in vpu bin filename if passed platform is
imx31ads since there is no VPU module in imx31ads.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 vpu/Makefile | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)

diff --git a/vpu/Makefile b/vpu/Makefile
index 4619a99..79316b8 100644
--- a/vpu/Makefile
+++ b/vpu/Makefile
@@ -11,7 +11,16 @@ BINPREFIX = vpu_fw_
 
 ifeq ($(PLATFORM), $(findstring $(PLATFORM), $(INCLUDE_LIST)))
 
-VERSION = $(shell echo $(PLATFORM) | awk '{print substr($$0, 1, 5)}' \
+# Same platform ID is used when building for imx31ads and imx32ads.
+# Change version number in bin filename to imx32 if passed platform is
+# imx31ads since there is no VPU module in imx31ads.
+ifeq ($(PLATFORM), IMX31ADS)
+PLATFORM_STR = IMX32ADS
+else
+PLATFORM_STR = $(PLATFORM)
+endif
+
+VERSION = $(shell echo $(PLATFORM_STR) | awk '{print substr($$0, 1, 5)}' \
 	| awk '{print  tolower($$0) }' )
 
 all: $(LIBNAME).so $(LIBNAME).a
-- 
1.8.0

